include/seqan/rna_io/dot_bracket_read_write.h

```diff
@@ -41,6 +41,7 @@
 #include <seqan/stream.h>
 #include <stack>
 #include <array>
+#include <cstddef>
```

Comment:
The header is included for the data type std::ptrdiff_t to store the distance of two pointers (see lines 163 & 172).